### PR TITLE
Feature/ci

### PR DESCRIPTION
**Pull request by Neurus1970**
Adding automated testing and continuous integration capabilities.

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sonar.projectKey=ReactiveODS
+sonar.projectName=Nuevo ODS MX
```
Comment (carlos-verdes) [inline sonar-project.properties]: I would change this to something more generic, what do you think?

Comment (Neurus1970) [inline sonar-project.properties]: I had no opinion about it, but (now) I'm thinking quite the opposite!

We need different names for different projects or else ...sonar wouldn't be able to show individual projects history. I've just crashed all previous history about the project for using the same name than before.

sonarqube

Comment (carlos-verdes) [inline sonar-project.properties]: My point was to use for this project Spark-akka-etl name ;)

Comment (Neurus1970) [inline sonar-project.properties]: Agreed. But I will preserve this name for this GitHub repo, and rename old project as POC. So new commits wouldn't inherit the POC reference.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        }
+
+        stage('Sonar Report') {
+            steps {
+                sh '/opt/sonar-runner/bin/sonar-runner'
+            }
+        }
+
+    }
+}
```
Comment (carlos-verdes) [inline Jenkinsfile]: Para la próxima... intentaría indentar sólo con 2 espacios y eliminar líneas que no aportan nada (como la 27)

**Comment by carlos-verdes** [comment]
LGTM and merging

**Pull request merged**